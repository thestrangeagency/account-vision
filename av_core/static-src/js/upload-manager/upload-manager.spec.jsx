import React from 'react';
import { shallow } from 'enzyme';

import UploadManager from './upload-manager';

jest.mock('../services/upload-api-service');

test('it renders', () => {
  const wrapper = shallow(<UploadManager year="2017" />);
  expect(wrapper).toMatchSnapshot();
});
