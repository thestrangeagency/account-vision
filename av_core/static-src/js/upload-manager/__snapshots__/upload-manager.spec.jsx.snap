// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="upload-manager mb-5"
>
  <Dropzone
    activeClassName="bg-light"
    className="dropzone p-5 my-5 border"
    disableClick={false}
    disablePreview={true}
    disabled={false}
    disabledClassName="bg-light"
    maxSize={Infinity}
    minSize={0}
    multiple={true}
    onDrop={[Function]}
    preventDropOnDocument={true}
  >
    <p
      className="m-0 text-center"
    >
      Drag files here or click to upload
    </p>
  </Dropzone>
  <UploadModal
    isSaveDisabled={false}
    onHidden={[Function]}
    onShown={[Function]}
    onSubmit={[Function]}
    setDescriptionRef={[Function]}
  />
  <DeleteConfirmationModal
    isActionDisabled={false}
    onClickConfirm={[Function]}
  />
</div>
`;
